appveyor.yml

```diff
@@ -2,7 +2,9 @@ environment:
   matrix:
   - julia_version: 1.0
   - julia_version: 1.2
+  - julia_version: 1.3 # single thread
   - julia_version: 1.3
+    JULIA_NUM_THREADS: 2 # two threads
   - julia_version: nightly
```

Comment:
(This file needs to be rebase to current master.) 

I think it is advisable to test this functionality in Julia 1.3 and beyond.

Reply:
Thanks, just rebased to current master

Reply:
... and added corresponding env variable settings for julia1.4 and nightly